dist/image-sequencer.js

```diff
@@ -47836,6 +47923,11 @@ function Run(ref, json_q, callback, progressObj) {
       var image = drawarray[pos].image;
       var i = drawarray[pos].i;
       var input = ref.images[image].steps[i - 1].output;
+
+      getStep = function getStep(offset) {
```

Comment:
Ah and also this is still a global, no? I'm reluctant about using a global. Can we stick with this.getStep or something.getStep to avoid this until we find a better solution? Thanks!!!

Reply:
@jywarren Actually I am pretty sure that if we want to call getStep() directly from inside module then either this function would have to be defined inside the module or we have to make it global, what that means for us is that if we want to do `getStep(offset)` then we have to use global or else we have to use `this.getStep()`

Reply:
Also @jywarren I don't think global here is a bad thing since utility functions are the only thing that we normally make global anyway

Reply:
Ok, so although I hear what you're saying, I really want this library to be self contained. Let's go for this.getStep and we can get this big fixed and next steps in another PR!

Reply:
@jywarren Okay doing this without global now😁